docs/docs/building.md

```diff
@@ -27,89 +26,63 @@ contract MyContract is Migratable {
 
-Register all the contract implementations of the first version of your
-project running:
+To deploy our app, we need to register the first version of our contract:
 
+```sh
+zos add MyContract
 ```
-zos add <contract_name_1>
-zos add <contract_name_2>
-...
-zos add <contract_name_n>
-```
-
-In our example, run:
 
-    zos add MyContract
+> **Note**: If you are working in a local development network like [Ganache](http://truffleframework.com/ganache/), you will need to [configure](http://truffleframework.com/docs/getting_started/project#alternative-migrating-with-ganache) your `truffle.js` file before running the `push` command.
 
-If you are working in a local development network like [Ganache](http://truffleframework.com/ganache/), you will need to [configure](http://truffleframework.com/docs/getting_started/project#alternative-migrating-with-ganache) your `truffle.js` file before running the `push` command.
+We can now push our application to the network by running:
 
-You can now push your application to the desired network by running:
-
-```
+```sh
 zos push --network <network>
 ```
 
-which will create a `zos.<network>.json` file with all the information specific to the chosen network. The format of this file is discussed in the [Advanced topics](advanced.md) section.
+This  will create a `zos.<network>.json` file with all the information specific to the chosen network. You can read more about this file in the [advanced topics](advanced.md#format-of-zosjson-and-zos-network-json-files) section.
 
-To create an upgradeable version of each of your contracts, run:
+To create an upgradeable version of our contract, we need to run:
 
-```
-zos create <contract_name_1> --network <network>
-zos create <contract_name_2> --network <network>
-...
-zos create <contract_name_n> --network <network>
+```sh
+zos create MyContract --init initialize --args 42 --network <network>
 ```
 
-This command takes an optional `--init` flag to call an initialization/migration
-function after you create the contract. In our simple example we want to pass a value (_e.g._: 42) to the initialize function:
+The `create` command takes an optional `--init` flag to call the initialization function after creating the contract, while the `--args` flag is to pass arguments to it. This way, we are initializing our contract with `42` as the value of the `x` state variable.
 
-```
-zos create MyContract --init initialize --args 42 --network development
-```
 
-After these simple steps, your upgradeable application is now on-chain!
+After these simple steps, our upgradeable application is now on-chain!
 
-## Upgrading your contracts
+## Upgrading our contract
 
-If, at a later stage, you want to upgrade your smart contracts' code in order to fix a bug or add a new feature, you can do it seamlessly using ZeppelinOS. 
+If, at a later stage, we want to upgrade our smart contract code in order to fix a bug or add a new feature, we can do it seamlessly using ZeppelinOS. 
```

Comment:
upgrade our smart contract code -> upgrade our smart contract

Reply:
It think "code" is fine, it helps focusing on the behavior and not the storage part of the upgrade.

Reply:
then it's "smart contract's code".